src/less/textbox/ds6/textbox.less

```diff
@@ -81,7 +81,7 @@ span.textbox__icon:first-child,
 // RTL
 
 [dir="rtl"] {
-    span.textbox__icon:first-child::before,
+    span.textbox__icon:first-child,
     .textbox__icon:first-child {
```

Comment:
Actually, just realizing we can probably drop the first of these grouped selectors, OR prefix the second selector with `svg` to make it explicit which tags we expect.